### ISSUE TITLE
Midterm Report Review -zw287

### ISSUE THREAD

**Issue by RobinWu218** [ZichengHu/ORIE-4741-Project#12]
Summary
The project group is trying to determine useful features in predicting Airbnb prices in New York City. They collected data from Kaggle and did extensive visualization to explore the datasets. In the midterm report, the group presented their findings and outlined their techniques in data preprocessing. Some initial modeling was also conducted, and they also pointed out future directions.

Likes:
1. I really like the data visualization presented in the report. They clearly demonstrate correlations among variables in the dataset. 
2. The log transformation was insightful. They clearly presented the reasoning behind their decision to use log transformation as the price distribution is right-skewed. 
3. Initially modelling with 5-fold cross validation was a solid first step. It shows that a simple linear model without feature engineering cannot fully capture the underlying distribution. It also serves as a baseline and motivation for future explorations.

Suggestions:
1. I would suggest that the group could try using l1 norm to do some feature selection before further feature engineering. It would be interesting to see if the result agrees with the correlation plot at the beginning of data visualization.
2. A possible next step would be to use neural networks or kernel methods. Since the underlying distribution seems to be non-linear, these methods could capture the non-linearity better.
3. There are also many other loss functions and regularizers to try. The current model uses quadratic loss and l2 norm. Based on what we learned in lecture, there are many other losses and regularizers the group can evaluate before exploring other non-linear methods.
